app/templates/views/about/about.html

```diff
@@ -0,0 +1,77 @@
+{% extends "base.html" %}
+
+{% set page_title = "About notify" %}
+
+{% block per_page_title %}
+{{page_title}}
+{% endblock %}
+
+{% block content_column_content %}
+<!-- {% with title=page_title %}{% include "components/best-practices/nav_breadcrumb.html" %}{% endwith %} -->
+<section class="usa-prose">
+  <h1>{{page_title}}</h1>
+  <p>Notify.gov is a text messaging service built by and for the government. We help agencies communicate more
+    effectively with the people they serve. With Notify.gov, federal and federally-funded programs can send customized
+    text messages that make it possible to:</p>
+  <ul>
+    <li>Meet people where they are</li>
+    <li>More effectively deliver program outcomes</li>
+    <li>Save administrative costs</li>
+    <li>Implement <a href="https://digital.gov/resources/delivering-digital-first-public-experience/"
+        target="_blank">21st Century IDEA</a> and other directives</li>
+  </ul>
+  <p>Notify.gov is an easy-to-use, web-based platform. It requires no technical expertise or system integration — users
+    can create an account and get started within minutes. We take the security and privacy of messaging data seriously
+    by minimizing data retention and using modern encryption methods.</p>
+  <!-- add youtube video component here -->
+  <h2 class="padding-bottom-2">Product Highlights</h2>
```

Comment:
Small change, but let's make this padding-bottom-3. In the future I'd like to write some styles to make the spacing/layout consistent throughout all the content pages without having to use too many utility classes.

Reply:
@jonathanbobel I made the padding change here but it's padding-bottom-2 for the guides pages. we may need to revisit padding on those h2 another time